app/src/main/java/woowacourse/movie/presentation/view/main/booklist/BookingListViewHolder.kt

```diff
@@ -1,29 +1,30 @@
 package woowacourse.movie.presentation.view.main.booklist
 
 import android.content.Intent
+import android.view.LayoutInflater
 import android.view.View
+import android.view.ViewGroup
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.example.domain.Reservation
 import woowacourse.movie.R
 import woowacourse.movie.presentation.view.main.home.bookcomplete.BookCompleteActivity
 
-class BookingListViewHolder(rootView: View) :
-    RecyclerView.ViewHolder(rootView) {
-    private val root = rootView
-    private val context = rootView.context
-    private val bookingDate = rootView.findViewById<TextView>(R.id.tv_booking_date)
-    private val bookingMovieTitle =
-        rootView.findViewById<TextView>(R.id.tv_booking_movie_title)
+class BookingListViewHolder(parent: ViewGroup) :
+    RecyclerView.ViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.item_booking_list, parent, false)) {
```

Comment:
```suggestion
    RecyclerView.ViewHolder(
        LayoutInflater.from(parent.context)
            .inflate(R.layout.item_booking_list, parent, false)
    ) {
```
ViewHolder에서 inflate하도록 잘 개선해주셨네요. 👍 
가로로 코드가 길어지게 되면, 개행을 해주는 것이 좋습니다.